packages/backend-lalr/data/HappyTemplate.hs

```diff
@@ -299,11 +259,9 @@ notHappyAtAll = Prelude.error "Internal Happy error\n"
 -----------------------------------------------------------------------------
 -- Hack to get the typechecker to accept our action functions
 
-#if defined(HAPPY_GHC)
 happyTcHack :: Happy_GHC_Exts.Int# -> a -> a
 happyTcHack x y = y
 {-# INLINE happyTcHack #-}
-#endif
```

Comment:
I think these lines should not have been removed; that would explain the CI errors.